Njord/src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -23,15 +38,56 @@ public class ElevatorSubsystem extends SubsystemBase {
   public enum ElevatorState {
     S_GoingUp, S_GoingDown, S_Stopped
     }
+
+  private final MutVoltage mVoltage = Volts.mutable(0);
```

Comment:
See a later comment. `mVoltage` is not needed because a conversion is not required.
```suggestion
```